<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>dzest</class>
 <widget class="QDialog" name="dzest">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>506</width>
    <height>282</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dzēst informāciju</string>
  </property>
  <widget class="QComboBox" name="dzestbildes">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>50</y>
     <width>151</width>
     <height>25</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>50</y>
     <width>151</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>Dzēst personas bildes</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>80</y>
     <width>151</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>Dzēst personu</string>
   </property>
  </widget>
  <widget class="QComboBox" name="dzestpersonas">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>80</y>
     <width>151</width>
     <height>25</height>
    </rect>
   </property>
  </widget>
  <widget class="QComboBox" name="dzestligumu">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>140</y>
     <width>151</width>
     <height>25</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>140</y>
     <width>151</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>Dzēst līgumu</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>110</y>
     <width>151</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>Dzēst adresi</string>
   </property>
  </widget>
  <widget class="QComboBox" name="dzestadreses">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>110</y>
     <width>151</width>
     <height>25</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_5">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>170</y>
     <width>151</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>Dzēst amatu</string>
   </property>
  </widget>
  <widget class="QComboBox" name="dzestamatu">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>170</y>
     <width>151</width>
     <height>25</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_6">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>200</y>
     <width>151</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>Dzēst piekļuvi</string>
   </property>
  </widget>
  <widget class="QComboBox" name="dzestpiekluvi">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>200</y>
     <width>151</width>
     <height>25</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="dzest_bildes">
   <property name="geometry">
    <rect>
     <x>350</x>
     <y>50</y>
     <width>121</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string>Dzēst bildes</string>
   </property>
  </widget>
  <widget class="QPushButton" name="dzest_personu">
   <property name="geometry">
    <rect>
     <x>350</x>
     <y>80</y>
     <width>121</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string>Dzēst personu</string>
   </property>
  </widget>
  <widget class="QPushButton" name="dzest_adresi">
   <property name="geometry">
    <rect>
     <x>350</x>
     <y>110</y>
     <width>121</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string>Dzēst adresi</string>
   </property>
  </widget>
  <widget class="QPushButton" name="dzest_ligumu">
   <property name="geometry">
    <rect>
     <x>350</x>
     <y>140</y>
     <width>121</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string>Dzēst līgumu</string>
   </property>
  </widget>
  <widget class="QPushButton" name="dzest_piekluvi">
   <property name="geometry">
    <rect>
     <x>350</x>
     <y>200</y>
     <width>121</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string>Dzēst piekļuvi</string>
   </property>
  </widget>
  <widget class="QPushButton" name="dzest_amatu">
   <property name="geometry">
    <rect>
     <x>350</x>
     <y>170</y>
     <width>121</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string>Dzēst amatu</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
